### PR TITLE
Notes for contributors and changes to readme

### PR DESCRIPTION
**Pull request by jameshensman**

### CODE REVIEW

**Comment by jameshensman** [comment]
@alexggmatthews @markvdw please add to these notes if there's anything missing or you feel should be amended.

File: contributing.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Keeping up with tensorflow
+GPflow tries to keep up with api changes in tensorflow as far as is reasonable, so that the latest GPflow will work with the latest (stable) tensorflow. Changing the version of tensorflow that we're compatible with requires a few tasks:
+ - update version used on travis via `travis.yml`
+ - update version used on codeship (requires codeship login)
+ - update `README.md`
+ - update version ussed by readthedocs.org via `docsrequire.txt`
+
+## Version numbering
+The main purpose of versioning GPflow is user convenience.
```
Comment (alexggmatthews) [inline contributing.md]: Might be good to add criteria for bumping version number?

File: contributing.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
+# Contributing to GPflow
+
+## Project scope
+We do welcome contributions to GPflow. However, the project is deliberately of limited scope, to try to ensure a high quality codebase: if you'd like to contribute a feature, please raise discussion via a github issue. 
+
+## Code Style
+ - Python code should follow the pep8 style. To help with this, we suggest using a plugin for your editor. 
+ - Practise good code as far as is reasonable. The Google [python syle guide](https://google.github.io/styleguide/pyguide.html) is a good place to look for more information, as is (we hope!) reading the existing GPflow code.
```
Comment (jameshensman) [inline contributing.md]: as far as I know, pep8 is the python style guide?

Comment (jameshensman) [inline contributing.md]: oh, reading that again you mean the google python style guide I think? Agree.

File: contributing.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+
+## Code Style
+ - Python code should follow the pep8 style. To help with this, we suggest using a plugin for your editor. 
+ - Practise good code as far as is reasonable. The Google [python syle guide](https://google.github.io/styleguide/pyguide.html) is a good place to look for more information, as is (we hope!) reading the existing GPflow code. 
+
+## Pull requests and the master branch
+All code that is destined for the master branch of GPflow goes through a PR. Only a small number of people can merge PRs onto the master branch (currently James Hensman, Alex Matthews, Mark van der Wilk and Alexis Boukouvalas). 
+
+## Tests and continuous integration
+GPflow is 99% covered by the testing suite. We changes to code to pass these tests, and for new code to be covered by new tests. Currently, tests are run by travis (python 3) and by codeship (python 2.7), coverage is reported by codecov.
```
Comment (jameshensman) [inline contributing.md]: fixed.

**Comment by jameshensman** [comment]
@alexggmatthews @markvdw I suggest merging this unless you disagree strongly with any of the guidance. Tweaks and additions can come later if needed.

**Comment by alexggmatthews** [comment]
Agree. I will merge if Mark is happy.

**Pull request merged**